javascript/src/components/navigation/HelpMenu.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {PropTypes} from 'react';
+import { NavDropdown, MenuItem } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+
+import Routes from 'routing/Routes';
+
+const HelpMenu = React.createClass({
+  propTypes: {
+    active: PropTypes.bool.isRequired,
+  },
+  render() {
+    return (
+      <NavDropdown navItem title="Help" id="user-menu-dropdown" active={this.props.active}>
+        <LinkContainer to={Routes.GETTING_STARTED}>
+          <MenuItem>Getting Started</MenuItem>
+        </LinkContainer>
+        <MenuItem href="http://docs.graylog.org" target="blank">
```

Comment:
Maybe this link should include the version of the documentation? Otherwise it will redirect to the documentation of Graylog 2.0.x, once it has been released, although people might still use Graylog 1.3.x.